wiki/misc/mappings/en.md

```diff
@@ -0,0 +1,99 @@
+but still aren't readable in english. Quilt uses Hashed Mojmap, in which every class, field, and method is prefixed by
+`C_`, `f_` and `m_` respectively, followed by an 8-letter hash. While developing mods, most of the time you'll see
+Fabric's intermediary instead, which uses `class_`, `field_` and `method_`, followed by a number. Mojang also publishes
+official mappings, often called Mojmap (short for Moj(ang)-map(pings)), for every version after 1.14, but, since they don't use
+intermediate mappings, you have to do some extra processing to use them where you'd use other mapping sets. Luckily, Loom
+does this process for you, so you don't have to worry about that and can easily replace the mappings used in your mod for
+Mojang's oficial ones.
+
+There a few different formats to store mappings, and as of the time this article was written, we are developing a new
+one with a bunch of improvements. In our ecosystem, the most used format is Tiny V2, which uses a single `.tiny` file to
+store mappings, and places fields and methods as "children" of their parent class. Another format we often use is the
+Enigma format, which uses a directory tree with a file for each top-level class, and organize the entries in a
+tree-like structure, placing each field, method and class as a child of another class or a top level one.
+
+## Editing your mappings
+
+Mapping sets don't have to contain a mapping for every class, field or method in a jar to be valid, and in fact most
+mapping sets aren't complete. For instance, Quilt Mappings (QM for short) has reached 99% completion at most. In a
+development environment, unmapped things will use their intermediary name instead of the obfuscated one, so if you have
+ever browsed Minecraft's code with QM or Fabric's Yarn applied, it's very likely you have seen quite a few intermediary
+names. It gets worse with method parameters: since they are really prone to incompatible changes between versions, they
+(usually) don't have intermediate names, thus the names you see in the code depend on the tooling that you used to
+decompile the game.
+
+If you want to add or change a wrong or bad name, or add documentation to the code, you can start by picking which
+mapping set you'll work on top of. In this article, we'll use QM, though the process for Yarn is almost identical.
+If you want to work on top of Mojang's mappings, you'll have to do some extra work which we won't cover here. If you
+are going to work on QM, we highly suggest taking a look at its [CONTRIBUTING.md](https://github.com/QuiltMC/quilt-mappings/blob/HEAD/CONTRIBUTING.md)
+and contributing your changes to the repository. You'll need some very basic Git knowledge, but it should be fairly
+easy if you've ever worked with Git before.
+
+To get started, clone or download [Quilt Mappings](https://github.com/QuiltMC/quilt-mappings) (if you want to
+contribute, [fork the repo] and clone it), and run `./gradlew mappings` in your command prompt or terminal. This will
```

Comment:
is there supposed to be a link on [fork the repo] ?

Reply:
Yep, forgot to add the link lol